tests/e2e/specs/editor/__snapshots__/publishingFlow.js.snap

```diff
@@ -2,6 +2,10 @@
 
 exports[`Publishing Flow should guide me towards creating a new post to embed my story 1`] = `
 "<!-- wp:web-stories/embed {\\"url\\":\\"http://localhost:8899/stories/publishing-flow-test\\",\\"title\\":\\"Publishing Flow Test\\",\\"poster\\":\\"\\"} -->
-<div class=\\"wp-block-web-stories-embed alignnone\\"><amp-story-player style=\\"width:360px;height:600px\\" data-testid=\\"amp-story-player\\"><a href=\\"http://localhost:8899/stories/publishing-flow-test\\">Publishing Flow Test</a></amp-story-player></div>
+<div class=\\"wp-block-web-stories-embed alignnone\\">
+			<amp-story-player style=\\"width: 360px; height: 600px; margin: 0\\" data-testid=\\"amp-story-player\\">
```

Comment:
Unrelated to this PR, but we should probably not save the `data-testid` attribute... Hmm.

Reply:
I didn't understand what that was for. I decided to leave it.